ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -255,9 +255,17 @@ const BaseVariantGene = React.memo(({ geneId, gene, variant, compact, showInline
       <GeneLinks>
         <a href={`https://decipher.sanger.ac.uk/gene/${gene.geneId}/overview/protein-genomic-info`} target="_blank">Decipher</a>
```

Comment:
Can we also add a popup hint for `Decipher`? Otherwise, it seems weird that it has nothing hovering while the other two buttons have.

Reply:
no, because the analysts don't have any confusion about that link and did not ask for a hover